open_street/osm_geometry.py

```diff
@@ -0,0 +1,76 @@
+import osmium
+from osmium.geom import GeoJSONFactory
+from typing import Any
+
+
+class GeometryRoadHandler(osmium.SimpleHandler):
+    def __init__(self) -> None:
+        osmium.SimpleHandler.__init__(self)
+        self.geometryDictionary: dict[str, str] = {}
+
+    def way(self, w: Any) -> None:
+        """This function gets called upon .apply_file for every road (OSM Way) that is in the file that apply_file is called with.
+
+        Args:
+            w (Any): OSM Way
+        """
+        if w.tags.get("highway") is not None:
+            try:
+                geo = GeoJSONFactory().create_linestring(w)  # Get the road linestring
+                self.geometryDictionary[w.id] = geo
+            except Exception as e:
+                print("error", e)
+                return
+
+
+def geometry_dictionary_to_geojson(geoDict: dict[str, str]) -> str:
+    """outputs geoJson formatted string from at osm_id -> linestring dictionary
```

Comment:
Sjovt at du starter med outputtet, jeg havde nok flippet det - "Convert geometryDictionary to geoJSON string"